reinforcement_learning/rl_cartpole_batch_coach/rl_cartpole_batch_coach.ipynb

```diff
@@ -0,0 +1,726 @@
+{
```

Comment:
<blockquote>Specify the choice of RL toolkit and framework. This automatically resolves to the ECR path for the RL Container.</blockquote><p>This is not applicable as we are using a custom image. </p><p> </p><blockquote>[Optional] Choose the metrics that you are interested in capturing in your logs. These can also be visualized in CloudWatch and SageMaker Notebooks. The metrics are defined using regular expression matching.</blockquote><p>I would suggest removing this point, or adding a cell that shows the metrics with regular expression. Right now it's not easy to understand what this point is talking about. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/awslabs/amazon-sagemaker-examples/pull/926/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='27'/>